src/homepageExperience/components/steps/Finish.tsx

```diff
@@ -79,7 +79,7 @@ export const Finish = (props: OwnProps) => {
       props.markStepAsCompleted()
       fireConfetti()
     }
-  }, [])
+  }, []) // eslint-disable-line react-hooks/exhaustive-deps
```

Comment:
I would avoid this if possible - it should be okay to put the props and wizard information in the dependency array, since none of that should change. Try removing the comment, and then testing the page with all of the dependencies React flags for you.